concordium-consensus/src/Concordium/GlobalState/Persistent/BlockState.hs

```diff
@@ -913,6 +913,72 @@ doGetActiveBakersAndDelegators pbs = do
                     ..
                 }
 
+doGetActiveDelegators
```

Comment:
I see that other functions in here also lack doc comments, but perhaps we should add some for these new ones?